api/src/main/java/org/apache/cloudstack/api/command/admin/user/UpdateUserCmd.java

```diff
@@ -66,7 +66,7 @@ public class UpdateUserCmd extends BaseCmd {
     @Parameter(name = ApiConstants.CURRENT_PASSWORD, type = CommandType.STRING, description = "Current password that was being used by the user. You must inform the current password when updating the password.", acceptedOnAdminPort = false)
     private String currentPassword;
 
-    @Parameter(name = ApiConstants.SECRET_KEY, type = CommandType.STRING, description = "The secret key for the user. Must be specified with userApiKey")
+    @Parameter(name = ApiConstants.USER_SECRET_KEY, type = CommandType.STRING, description = "The secret key for the user. Must be specified with userApiKey")
```

Comment:
@shwstppr 
IMHO this should not be changed

Reply:
@weizhouapache only the variable name is changed (adjusted to actual string value). API parameter is still the same - `usersecretkey`
https://github.com/apache/cloudstack/pull/8674/files#diff-72d5bf21c12ffd0a3c2d3a6033ec70ae5ba6c31338e915fe65df1b7a67827a9eL403

https://github.com/apache/cloudstack/pull/8674/files#diff-72d5bf21c12ffd0a3c2d3a6033ec70ae5ba6c31338e915fe65df1b7a67827a9eR491

Reply:
ok @shwstppr 
double-checked it, it is the same as before
was a bit confused.
sorry